app/Models/InfoLocation.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class InfoLocation extends Model
+{
+    protected $table = 'info_locations';
+
+    protected $fillable = [
+        'user_id',
+        'name',
+        'image',
+        'language',
+        'position',
+        'seat_id',
+        'working_time'
+    ];
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Models/InfoLocation.php</br>Comma required after last value in array declaration